app/javascript/modules/listings/DirectoryHelpers.tsx

```diff
@@ -226,8 +227,12 @@ export const getListingCards = (listings, directoryType, stackedDataFxn, hasFilt
     )
   })
 
-export const openListingsView = (listings, directoryType, stackedDataFxn, filtersSet?) =>
-  listings.length > 0 && getListingCards(listings, directoryType, stackedDataFxn, filtersSet)
+export const openListingsView = (
+  listings: RailsListing[],
+  directoryType: DirectoryType,
+  stackedDataFxn: (listing: RailsRentalListing) => Record<string, StackedTableRow>[],
```

Comment:
question: I'm seeing `stackedDataFxn` appear a lot in a other places as an untyped argument. Is it worth the effort to define this type in a more accessible place so that we can use it in more places?